list-sensors.cpp

```diff
@@ -10,128 +12,245 @@
 #include <sdbusplus/exception.hpp>
 #include <string>
 
+// Bus handler singleton
+static sdbusplus::bus::bus systemBus = sdbusplus::bus::new_default();
+
+using PropertyValue = std::variant<int64_t, std::string, bool>;
+using PropertyName = std::string;
+using PropertiesMap = std::map<PropertyName, PropertyValue>;
+
+constexpr auto COLOR_RED = "\033[31;1m";
```

Comment:
We should avoid xterm codes to support any output devices. I would remove colors from here.

Reply:
done